aries-site/src/examples/templates/forms/managing-child-objects/CreateCluster.js

```diff
@@ -0,0 +1,243 @@
+import { useState } from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Heading,
+  RangeInput,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+import { ButtonGroup } from '../../ButtonGroup';
+import { FormChildObjects } from '../../FormChildObject';
+
+const hostTemplate = {
+  name: '',
+  host: '',
+  cpu: '',
+  memory: '',
+};
+
+const INPUT_MAP = {
+  name: ({ key, index, ...rest }) => (
+    <FormField
+      key={key}
+      htmlFor={`hosts[${index}].name`}
+      name={`hosts[${index}].name`}
+      label="Host name"
+    >
+      <TextInput
+        id={`hosts[${index}].name`}
+        name={`hosts[${index}].name`}
+        {...rest}
+      />
+    </FormField>
+  ),
+  host: ({ key, index, ...rest }) => (
+    <FormField
+      key={key}
+      htmlFor={`hosts[${index}].host`}
+      name={`hosts[${index}].host`}
+      label="Host address"
+      required
+      aria-required="true"
+    >
+      <TextInput
+        id={`hosts[${index}].host`}
+        name={`hosts[${index}].host`}
+        {...rest}
+      />
+    </FormField>
+  ),
+  cpu: ({ key, index, ...rest }) => (
+    <Box key={key} width="xsmall">
+      <FormField
+        htmlFor={`hosts[${index}].cpu`}
+        name={`hosts[${index}].cpu`}
+        label="CPU cores"
+        required
+        aria-required="true"
+      >
+        {/* Make this a masked input */}
+        <Select
+          id={`hosts[${index}].cpu`}
+          name={`hosts[${index}].cpu`}
+          options={[2, 4, 6, 8]}
+          {...rest}
+        />
+      </FormField>
+    </Box>
+  ),
+  memory: ({ key, index, ...rest }) => (
+    <Box key={key} width="xsmall">
+      <FormField
+        key={key}
+        htmlFor={`hosts[${index}].memory`}
+        name={`hosts[${index}].memory`}
+        label="Memory (GB)"
+        required
+        aria-required="true"
+      >
+        <Select
+          id={`hosts[${index}].memory`}
+          name={`hosts[${index}].memory`}
+          options={[32, 64, 128, 256, 512]}
+          {...rest}
+        />
+      </FormField>
+    </Box>
+  ),
+};
+
+const defaultValues = {
+  'cluster-name': '',
+  'resource-manager': false,
+  'automation-level': '',
+  'migration-threshold': 15,
+  hosts: [{ ...hostTemplate }],
+};
+
+export const CreateCluster = () => {
+  const [formValues, setFormValues] = useState(defaultValues);
+
+  const handleAdd = () => {
```

Comment:
maybe add comments on the `add` `remove`

Reply:
Done.